_posts/2024-06-03-so-you-want-better-opensearch-documentation.markdown

```diff
@@ -0,0 +1,60 @@
+
+Charlie then asked the audience what their thoughts were and we captured a lot of great insights. I then followed up with Heather Halter whose remarks are in *italics*. 
+
+**“Older versions of OpenSearch docs often show up in Google which is often jarring. Can we make sure Google is properly indexing the latest content?“**
+
+_While we’ve made a lot of progress in this area, particularly with implementing a redirect strategy, we are currently planning on adding canonical links to all the older pages. The goal is to get this work done by the end of Q2, which should show a big improvement in the back half of the year._
+
+**“I often find a GitHub issue that answers my question and I ⭐it.” Someone suggested that we look for comments on GitHub issues that have lots of stars and use that as input to where docs could be improved.**
+
+ _Great suggestion!_
+
+**“My 2-yr old won’t let me work at night” was one response to the question of why people who find a gap in documentation don’t pursue opening a PR to improve the docs.** 
+
+_We love community contributions and have made it really easy to start. Check out the [CONTRIBUTING.md](http://contributing.md/) file and watch this [YouTube video](https://www.youtube.com/watch?v=zmVC3fQwnVI) on our documentation process and how you can contribute. You can also search through the issues labeled "[good first issue"](https://github.com/opensearch-project/documentation-website/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) or "[help wanted"](https://github.com/opensearch-project/documentation-website/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22)._
+
+**Backwards compatibility in API’s came up as an issue. It would be interesting if we could better communicate API changes over time.** 
+
+_We use the "Introduced (version)" label to indicate the release in which a feature is introduced. But going backwards to identify everything is a big job. We received a suggestion to do this for field types, so I labeled it with “*[*_good first issue_*](https://github.com/opensearch-project/documentation-website/issues/6993)*" in case someone wants to jump in._
+
+**Someone did make the comment that they did NOT believe that developers don’t like to write docs, especially in open source, but that we need to strive to continue to lower the barriers to contributing docs.** 
+
+_We are relying more and more on developers to write content and are looking at tools that make it easier and faster to write good documentation. We also have a great automated style checker built into our process that helps developers learn how to write better by correcting their mistakes._
+
+**One person asked what powered the site search, and the response that it is OpenSearch sparked a conversation about could we apply the new capabilities in OS like hybrid search and vectors to what powers the website. Kris Freedain spoke up with a “Who wants to collaborate on this?” and at least one person raised their hand ;-).**
```

Comment:
Should we link Kris' name to his GitHub profile? It looks as though we link to his LinkedIn below, but is that the preferred method of getting in contact?

Reply:
This is a great question. Since I do have a 'Community Member' profile on opensearch.org - it would make sense to link to it.  https://opensearch.org/community/members/kris-freedain.html 

It contains 4 links where the audience would be able to properly contact me. (LinkedIn, GitHub, X/Twitter, and Mastodon) 

I think this would be best for general guidance if this question comes up again. 
If the individual named in a future post does not have a Community Member profile, we could leave it up to them (linking to their LinkedIn, GitHub, X/Twitter, or Mastodon) - and possibly email.